versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    mongodbatlas = {
+      source = "mongodb/mongodbatlas"
+    }
+  }
+  required_version = ">= 0.13"
+}
```

Comment:
I was not sure which version should I put here.

Reply:
In terms of the terraform version our provider currently supports up to 1.3 and above (https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs#hashicorp-terraform-versionhttpswwwterraformiodownloadshtml-compatibility-matrix). I believe for modules we can define the terraform version constraint to 1.0 and above.
For the case of the provider version it be harder to identify what is the oldest version that can create these set of resource with issues. For now we can leave without a constraint, and if issues appear related to older versions we can define the constraint.